examples/hybrid/driver.jl

```diff
@@ -74,6 +74,8 @@ additional_cache(Y, params, dt; use_tempest_mode = false) = merge(
     isnothing(microphy) ? NamedTuple() : zero_moment_microphysics_cache(Y),
     isnothing(forcing) ? NamedTuple() : held_suarez_cache(Y),
     isnothing(rad) ? NamedTuple() :
+        rad == "gray" ?
+        rrtmgp_model_cache(Y, params; radiation_mode = GrayRadiation()) :
         rrtmgp_model_cache(Y, params; idealized_h2o),
```

Comment:
@dennisYatunin Could you help make this better? It will get more complicated once we have more options for radiation.

Reply:
never mind I asked Charlie to help:)